cross/rackspace-novaclient/Makefile

```diff
@@ -0,0 +1,15 @@
+PKG_NAME = rackspace-novaclient
```

Comment:
I'd suggest to update pip, setuptools and wheel to latest. There was a fix for this type of packages in wheel 0.25: https://bitbucket.org/pypa/wheel/commits/985dbf6d1692980d6f393f2db317f855b6713e6d